cdap-ui/app/cdap/components/DataPrep/DataPrepServiceControl/ServiceEnablerUtilities.js

```diff
@@ -138,8 +138,7 @@ export default function enableDataPreparationService(shouldStopService) {
     function ping() {
       MyDataPrepApi.ping({ namespace })
         .subscribe(() => {
-          observer.onCompleted();
-          window.location.reload();
```

Comment:
So two things,
1. If we are forcing the user to upgrade to latest wrangler version (since the new UI is not compatible with the old wrangler) would this still be the same flow?
2. If we still need a blocking step that forces the user to upgrade wrangler then lets add it in another PR. Lets keep this PR for integration and have another PR for upgrade.